client-app/pages/company/info.vue

```diff
@@ -29,12 +27,49 @@
             </template>
           </VcInput>
         </div>
+      </VcWidget>
```

Comment:
Add empty lines to visually separate tags.

Reply:
fixed